docker/django.dockerfile

```diff
@@ -29,6 +29,9 @@ LABEL org.label-schema.schema-version="1.0" \
     com.cossacklabs.docker.container.build-date=$BUILD_DATE \
     com.cossacklabs.docker.container.type="product"
 
+# Fix CVE-2019-5021
+RUN echo 'root:!' | chpasswd -e
```

Comment:
now alpine:3.9 have fix for this vulnerability )

Reply:
Yes, I know. But keeping in mind that bug, I'd prefer to set this explicitly. ;-)